ArkKit/ArkActionContext.swift

```diff
@@ -1,4 +1,8 @@
+/**
+ * A facade which encapsulates all the sub-contexts used in the execution of an action in `ArkBlueprint`.
+ */
 struct ArkActionContext {
```

Comment:
Think we can rename to `ArkContext` instead.

`ArkActionContext` seems to have similar nomenclature with `Rules.Action`